pam/update.sls

```diff
@@ -2,6 +2,10 @@
 
 {% if grains.os_family == 'Debian' %}
 pam-auth-update:
+  module.run:
```

Comment:
What is difference or benefit of using `module.run.file.remove` over `salt.states.file.absent`.  I never use `module.run` so just wondering?

Reply:
If it works, then its fine. thanks.